CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+## 1.10.0-beta.1
+
+### ✨ Features
+
+* Add `mapboxgl.preloadWorkerPool()` and `mapboxgl.removePreloadedWorkerPool()` methods to allow developers to optimize load times for their maps ([#9391](https://github.com/mapbox/mapbox-gl-js/pull/9391))
+* Add `index-of` and `slice` expressions to search arrays and strings for the first occurrence of a specified value and return a section of the original array or string ([#9450](https://github.com/mapbox/mapbox-gl-js/pull/9450)) (h/t [lbutler](https://github.com/lbutler))
+* Correctly set RTL text plugin status if the plugin URL could not be loaded. This allows developers to add retry logic on network errors when loading the plugin ([#9489](https://github.com/mapbox/mapbox-gl-js/pull/9489))
+
+### 🍏 Improvements
+
+* An outstanding list of gesture issues and improvements were addressed in this release ([#9365](https://github.com/mapbox/mapbox-gl-js/pull/9365))
```

Comment:
nit: HMmmm which meaning of "outstanding" are we trying to convey? Its presence strikes me as slightly off

Reply:
Also: do we think it's necessary to have this much detail in the changelog, especially when our changes are in accordance with general widely-adopted best practices for gestures? I lean towards a higher level summary and then a link to the PR for the rest.

Reply:
Removed and focused on smaller set instead https://github.com/mapbox/mapbox-gl-js/pull/9542/commits/f5aa3f619aa1baa99f4a5ade94f28a8b3ec7e1d5.

@ansis Please confirm that this reflects your work appropriately.